internal/tsc_wrapped/compiler_host.ts

```diff
@@ -199,7 +201,9 @@ export class CompilerHost implements ts.CompilerHost, tsickle.TsickleHost {
    * @return the path without any rootDirs, eg. path/to/file
    */
   private rootDirsRelative(fileName: string): string {
-    for (const root of this.options.rootDirs) {
+    const rootDirs = this.options.rootDirs;
```

Comment:
nit: why bother declaring a local variable here?

Reply:
Just being explicit that there's no function changes here by using the variable in the same way.
Happy to refactor if you like

Reply:
Oh wait, my bad, that statement applies to all other for loop replacements.
This instance was to avoid going through `this` and `options` on each access, which can be slow depending on if it's a hot path and if `options` is mono or mega morphically tracked. So this is the safe option
Some notes here on the topic: https://github.com/angular/angular/blob/master/packages/core/src/render3/PERF_NOTES.md#monomorphic-vs-megamorphic-code